docs/README.md

```diff
@@ -40,22 +34,28 @@ functionality for docs, so it's largely optional. Requirements:
    Set a system environment variable named `PLANTUML_JAR` with a value of the fullpath
```

Comment:
Should the environment variable name be updated to be `DOXYGEN_PLANTUML_JAR_PATH` to match the "Configuration" section below?

Reply:
Good catch, thank you. Done.

Reply:
You're welcome!